packages/dd-trace/src/priority_sampler.js

```diff
@@ -117,6 +125,13 @@ class PrioritySampler {
     context._sampling.mechanism = mechanism
 
     const root = context._trace.started[0]
+
+    if (process.env.DD_PRIORITY_SAMPLER_TRACKING === 'true') {
```

Comment:
This environment variable is no longer needed as we have moved to `log.trace` instead.